source/magnifier/__init__.py

```diff
@@ -0,0 +1,209 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2025-2025 NV Access Limited, Antoine Haffreingue
+# This file may be used under the terms of the GNU General Public License, version 2 or later, as modified by the NVDA license.
+# For full terms and any additional permissions, see the NVDA license file: https://github.com/nvaccess/nvda/blob/master/copying.txt
+
+"""
+NVDA Magnifier module.
+Handles module initialization, configuration and settings interaction.
+"""
+
+import config
+
+from .fullscreenMagnifier import FullScreenMagnifier, FullScreenMode
+from .utils.filterHandler import filter
+
+_magnifier: FullScreenMagnifier | None = None
+
+# Configuration specification
+confspec = {
+	"magnifier": {
+		"defaultZoomLevel": "float(min=1.0, max=10.0, default=2.0)",
+		"defaultFullscreenMode": "string(default='center')",
+		"defaultFilter": "string(default='normal')",
+		"keepMouseCentered": "boolean(default=False)",
+		"saveShortcutChanges": "boolean(default=False)",
+	}
+}
```

Comment:
while I like this idea of adding the config spec separately a lot, I'm concerned that moving config out of configSpec.py might introduce bugs.
The order of loading is very important at start up, and if the config for magnifier is not loaded with the rest of the config, it could cause issue.